content/kubermatic/main/installation/upgrading/upgrade-from-2.26-to-2.27/_index.en.md

```diff
@@ -0,0 +1,164 @@
+
+## Pre-Upgrade Considerations
+
+{{% notice warning %}}
+Please review [known issues]({{< ref "../../../architecture/known-issues/" >}}) before upgrading to understand if any issues might affect you.
+{{% /notice %}}
+
+KKP 2.27 adjusts the list of supported Kubernetes versions and removes support for Kubernetes 1.28. Existing user clusters need to be migrated to 1.29 or later before the KKP upgrade can begin.
+
+### Removal of CentOS Support
+
+CentOS has reached its End of Life (EOL), and as a result, KKP 2.27 has removed support for CentOS operating systems. Ensure that all user clusters are migrated to a supported operating system before proceeding with the upgrade.
+
+### VSphere Credentials Handling
+
+In KKP 2.27, VSphere credentials are now handled properly. For existing user clusters, this change affects the credentials in `machine-controller` and `Operating System Manager (OSM)`. Specifically, the credentials will switch to using `infraManagementUser` and `infraManagementPassword` instead of the previous `username` and `password` when specified. Ensure that your configurations are updated accordingly to prevent any disruptions.
+
+### OpenStack Floating IP Pool Fix
+
+A regression in KKP v2.26.0 caused the floatingIPPool field in OpenStack clusters to be overridden with the default external network.
+
+If your OpenStack clusters use a floating IP pool other than the default, you may need to manually update Cluster objects after upgrading to v2.27.
+
+* Action Required:
+  *  After the upgrade, check your OpenStack clusters and manually reset the correct floating IP pool if needed.
+  *  Example command to check the floating IP pool
+  ```sh
+  kubectl get clusters -o jsonpath="{.items[*].spec.cloud.openstack.floatingIPPool}"
+  ```
+  * If incorrect, manually edit the Cluster object:
+  ```sh
+  kubectl edit cluster <cluster-name> 
+  ```
+  
+### Velero Configuration Changes
+
+By default, Velero backups and snapshots are turned off. If you were using Velero for etcd backups and/or volume backups, you must explicitly enable them in your values.yaml file.
+
+```yaml
+velero:
+  backupsEnabled: true
+  snapshotsEnabled: true
+```
+Additionally, the node-agent daemonset is now disabled by default. If you were using volume backups (velero.snapshotsEnabled: true), you also need to enable it. This ensures volume backups function as expected.
+
+```yaml
+velero:
+  deployNodeAgent: true
+```
+
+### K8sgpt-operator
+
+K8sgpt-operator has been introduced to replace the now deprecated `k8sgpt(non-operator)` application. K8sgpt application will be removed in the future releases.
+
+### Environment Variable Change for Equinix Metal
+
+KKP now uses `METAL_` environment variables instead of `PACKET_` for machine-controller and KubeOne. Ensure any configurations referencing PACKET_ variables are updated accordingly.
```

Comment:
Please add something to the effect of "In general this should not require any actions on the administrator part."